Makefile

```diff
@@ -369,13 +375,17 @@ ifneq (,$(findstring "$(PLATFORM)", "linux" "gnu_kfreebsd" "kfreebsd-gnu" "gnu")
     OPTIMIZE = $(OPTIMIZEVM) -ffast-math
     HAVE_VM_COMPILED=true
   else
-  ifeq ($(ARCH),ppc)
-    ALTIVEC_CFLAGS = -maltivec
+  ifneq ($(findstring $(ARCH),ppc ppc64 ppc64le),)
     HAVE_VM_COMPILED=true
   endif
+  ifeq ($(ARCH),ppc)
+    OPTIMIZE += -mcpu=powerpc
+  endif
   ifeq ($(ARCH),ppc64)
-    ALTIVEC_CFLAGS = -maltivec
```

Comment:
There is no need to explicitly declare `-maltivec` because `-mcpu=power7` would set that automatically for us